code/lib/node-logger/package.json

```diff
@@ -24,13 +24,11 @@
     ".": {
       "types": "./dist/index.d.ts",
       "node": "./dist/index.js",
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs"
+      "require": "./dist/index.js"
```

Comment:
I have a setup where I need the "import" declaration (nx / vite) -- in this case, adding back `"import": "./dist/index.js"` would solve my issue while still using a "prebundled" version of node-logger?

This broke our upgrade from storybook 7.0.24 to 7.4.46